nvflare/private/fed/client/client_json_config.py

```diff
@@ -141,7 +139,7 @@ def finalize_config(self, config_ctx: ConfigContext):
         )
 
         ConfigService.initialize(
-            section_files={SystemConfigs.APPLICATION_CONF: os.path.basename(self.config_files[0])},
+            section_files={},
```

Comment:
Do we need this "section_files" arguments in "ConfigService" anymore?
Should we remove it and related get/set sections in follow PRs?

Reply:
Before making further changes, let's discuss our overall approach.